syntaxes/idris.tmLanguage

```diff
@@ -121,6 +121,12 @@
 			</array>
 		</dict>
 		<dict>
+			<key>match</key>
+			<string>(?&lt;!\w)\'\w++(?!')</string>
```

Comment:
Can you give me some examples of this syntax change? I just can not get my sense to embrace this change actually 😅.